lib/Models/ChartDataGroup.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+/*global require*/
+var clone = require("terriajs-cesium/Source/Core/clone");
+var defineProperties = require("terriajs-cesium/Source/Core/defineProperties");
+var freezeObject = require("terriajs-cesium/Source/Core/freezeObject");
+
+var CatalogGroup = require("./CatalogGroup");
+var inherit = require("../Core/inherit");
+
+/**
+ * A {@link CatalogGroup} representing a collection of layers for charting
+ *
+ * @alias ChartDataGroup
+ * @constructor
+ * @extends CatalogGroup
+ *
+ * @param {Terria} terria The Terria instance.
+ */
+var ChartDataGroup = function(terria) {
```

Comment:
Do you need a separate type for this, versus a regular `CatalogGroup` with a known `id`?

Reply:
I thought early on it would have more chart specific behaviour, but it does not so I've reverted it back to a `CatalogGroup` for now 👍